tests/cucumber.conf.js

```diff
@@ -0,0 +1,27 @@
+const { Before, BeforeAll, AfterAll, After, setDefaultTimeout } = require("@cucumber/cucumber");
+const { chromium } = require("playwright");
+require('dotenv').config();
+
+const timeout = process.env.TIME_OUT ? parseInt(process.env.TIME_OUT) : 60000;
```

Comment:
`TIME_OUT` can be too general. `setDefaultTimeout` sets the timeout for async hooks and steps (see:  [setDefaultTimeout](https://github.com/cucumber/cucumber-js/blob/main/docs/support_files/timeouts.md))
```suggestion
const timeout = process.env.ASYNC_TIMEOUT ? parseInt(process.env.ASYNC_TIMEOUT) : 60000;
```